lib/unread/readable.rb

```diff
@@ -124,9 +125,14 @@ def mark_as_read!(options)
       end
 
       def read_mark(reader)
+        assert_reader(reader)
```

Comment:
This function is undocumented (see #89) and untested. However since `read_at` uses it and the other methods use `assert_reader` one way or the other, I decided to add it here as well.